jclouds-plugin/src/main/resources/jenkins/plugins/jclouds/compute/JCloudsCloud/config.jelly

```diff
@@ -20,11 +20,8 @@
     <f:entry title="Credential" field="credential">
         <f:password/>
     </f:entry>
-    <f:entry title="RSA Private Key" field="privateKey">
-        <f:textarea/>
-    </f:entry>
-    <f:entry title="Public Key" field="publicKey">
-        <f:textarea/>
+    <f:entry title="${%Cloud RSA key}" field="cloudGlobalKeyId">
```

Comment:
Actually it is in `jclouds-plugin/src/main/resources/jenkins/plugins/jclouds/compute/JCloudsCloud/help-privateKey.html`, it needs to be renamed to match the new filed name.